apps/dashboard/src/routes/org/[slug]/pathways/[id]/certificates/+page.svelte

```diff
@@ -0,0 +1,5 @@
+<script></script>
+
+<section class="border-2 border-black flex justify-center items-center w-full h-full">
```

Comment:
There is a component called `CourseContainer` which is used by courses to wrap each route within `courses/[id]/*`.

Please create something like that for Pathways and use around all its nested routes

Reply:
@tunny17 what's the update with this one?